site/content/contribute/desktop/code-signing.md

```diff
@@ -0,0 +1,188 @@
+---
+title: "Code Signing"
+date: 2018-01-16T10:32:51-05:00
+subsection: "Desktop App"
+---
+# Code Signing
+When releasing the Mattermost Desktop application for Windows and macOS, we have to sign the executable with a certificate that allows the end users' computer to verify our identity. 
+
+The signing procedure varies depending on the platform that the release is destined for, and the platform that the signing operation is performed on. This page attempts to document the procedure and some common pitfalls that developers may encounter along the way.
+
+## Code Signing Windows Releases
+Releases destined for Windows can be code signed on Windows, macOS, or Linux. 
+
+#### Prerequisites
+In order to code sign releases on behalf of Mattermost Inc., you'll need a `.pfx` (Personal Information Exchange) file that contains Mattermost's public key (SSL certificate file), and the associated private key file. The file is protected by a password that you'll need in order to use it. 
+
+This file has been shared in LastPass. Talk to Joram Wilander, Corey Hulen, or Jonathan Fritz to get access to it. 
+
+#### Code Signing on Windows
+On Windows hosts, Microsoft's [`signtool`](https://msdn.microsoft.com/en-us/library/windows/desktop/aa387764(v=vs.85).aspx) utility can be used to code sign releases.
```

Comment:
@jwilander if you've done this in the past, feel free to fill in some documentation here.

Reply:
Added a bit more info